nixos/modules/misc/ids.nix

```diff
@@ -347,6 +347,7 @@ in
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
       # shadow = 318; # unused
+      plikd = 319;
```

Comment:
[Don't assign static ids](https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md). I think `DynamicUser` would be a good fit here.